server/aesdsocket-start-stop.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh 
```

Comment:
@prvbharadwaj1731 don't check your `server/aesdsocket` file into your repo, this can cause problems with buildroot build.  You need to ensure you are including the file built using buildroot.